cdc/owner.go

```diff
@@ -384,7 +430,26 @@ func (o *ownerImpl) loadChangeFeedInfos(ctx context.Context) error {
 		var exist bool
 
 		if cfInfo, exist = o.changeFeedInfos[changeFeedID]; exist {
+			for cid, pinfo := range etcdChangeFeedInfo {
+				if _, ok := cfInfo.processorLastUpdateTime[cid]; !ok {
+					cfInfo.processorLastUpdateTime[cid] = time.Now()
+					continue
+				}
+
+				oldPinfo, ok := cfInfo.ProcessorInfos[cid]
+				if !ok || oldPinfo.ResolvedTs != pinfo.ResolvedTs {
```

Comment:
both `ResolvedTs` and `CheckpointTs` change can trigger the update of `last update time`?

Reply:
only check ResolvedTS here do you means check it both?

Reply:
`if !ok || oldPinfo.ResolvedTs != pinfo.ResolvedTs || oldPinfo.CheckpointTs != pinfo.CheckpointTs`
like this?